hieradata/clients/azure.ci.jenkins.io.yaml

```diff
@@ -284,6 +284,35 @@ profile::jenkinscontroller::jcasc:
           virtualNetworkResourceGroupName: "prod-jenkins-public-prod"
           subnetName: "ci.j-agents-vm"
           spot: true
+        - name: "ubuntu-20-test"
+          description: "Ubuntu 20.04 LTS test"
+          imageDefinition: jenkins-agent-ubuntu-20.04
+          os: "ubuntu"
+          storageAccount: ENC[PKCS7,MIIBiQYJKoZIhvcNAQcDoIIBejCCAXYCAQAxggEhMIIBHQIBADAFMAACAQEwDQYJKoZIhvcNAQEBBQAEggEAXcLW9OnUwAhARiFS6P8vg8YrfYj9DHXTJmXqp+U/Ytjeova0bH/C8bhEbOykV4nJLMReHrrfEu5Jx+Eg+wjfq7LGD4bkAsp3covik/lkxAEDiMACIU3mWGlgeQ+0Tf4tpEHDOlWXNiA33T+3Knr1/v4H4vQOWC63tASAUUIPys2sesrv4RilEqKTd39oT9ugsQDVhftEHbp4eaqmI9zTXpU5hvy2fpJp5F2b5o7ohNvdRsovrsbN7XWJvMxvRbg/YMK/yq3zdowBzUFqXRcmiNC7T2iV0/lNjKKJOV1j4y0VDQoMomOZ5+zrozDq44FhwtAXRHJMyHr8lhtYZth0PTBMBgkqhkiG9w0BBwEwHQYJYIZIAWUDBAEqBBBVFqN/yr2UmCADos0nCHO7gCC7w2deUgakcUIXAZ7Y1M+Y3+dGENY/F2cNkceceJpQhw==]
+          location: "East US 2"
+          instanceType: Standard_D4s_v3 # 4 vCPUS / 16 Gb
+          architecture: amd64
+          labels:
+            - test-datadog
+          maxInstances: 1
+          useAsMuchAsPosible: false
+          credentialsId: "jenkinsvmagents-userpass"
+          usePrivateIP: true
+          virtualNetworkName: "prod-jenkins-public-prod"
+          virtualNetworkResourceGroupName: "prod-jenkins-public-prod"
+          subnetName: "ci.j-agents-vm"
+          spot: true
+          executeInitScriptAsRoot: true
+          initScript:
```

Comment:
initScript is a string not an array

see example:
https://github.com/hmcts/cnp-flux-config/blob/866ffa780479849229dafc4fdeae39d0a5de2214/apps/jenkins/jenkins/ptl-intsvc/jenkins.yaml#L543-L565

Reply:
Yes, in jcasc :) here you have hieradata. Alas the yaml ruby parser does not manage efficiently the multiline strings so we used array of string for this . Template then iterates on the array to make the multiline block in jcasc final

Reply:
I see odd

Reply:
No problem, thanks for post-checking: it is healthy and ensure that we do not miss anything!